data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -161,8 +169,10 @@
 ]
 
 FITBIT_CLEANING_CLASSES = [
-    (RemoveFitbitDataIfMaxAgeExceeded,),
-    (PIDtoRID,),
+    RULE_ARGS(RemoveFitbitDataIfMaxAgeExceeded, ['mapping_dataset_id']),
```

Comment:
Right now, yes.  Because as of now, this is a combined dataset.  But, in the future, this can be updated to just use the existing deid_mapping table (so you don't have to recreate that information).